src/lib/storage/run_length_column/run_length_column_iterable.hpp

```diff
@@ -0,0 +1,134 @@
+#pragma once
```

Comment:
Where is the documentation for RLE?

Reply:
We don’t have a documentation for dictionary encoding either.

Reply:
We should have. At least the memory layout for dictionary encoding is easier to understand. A short explanation / example of how a couple of values are stored in the RLE column would be sufficient.

Reply:
done.